test/testCandidateSplitting.cpp

```diff
@@ -0,0 +1,90 @@
+#include "crpropa/Units.h"
+#include "crpropa/Common.h"
+#include "crpropa/ParticleID.h"
+#include "crpropa/module/CandidateSplitting.h"
+
+#include "gtest/gtest.h"
+#include <stdexcept>
+#include <cmath>
+
+namespace crpropa {
+
+TEST(testCandidateSplitting, SimpleTest) {
+
+
+	int n_split = 2;
+	int n_bins = 4;
+	double minWeight = pow(1./n_split, 2);
+	double Emin = 1*GeV;
```

Comment:
I would change the Energy to higher values, as 1 GeV for protons is not the regime where CRPropa can be used safely.

Reply:
Actually, it doesn't matter for the test, it's dimensionless now.